android/src/com/frostwire/android/gui/adapters/TransferListAdapter.java

```diff
@@ -318,9 +318,10 @@ private String populateBittorrentDownloadMenuActions(BittorrentDownload tag, Lis
             } else {
                 boolean wifiIsUp = NetworkManager.instance().isDataWIFIUp();
                 boolean bittorrentOnMobileData = ConfigurationManager.instance().getBoolean(Constants.PREF_KEY_NETWORK_USE_MOBILE_DATA);
+                boolean bittorrentOnlyWifi = ConfigurationManager.instance().getBoolean(Constants.PREF_KEY_NETWORK_USE_WIFI_ONLY);
```

Comment:
or i guess another way to go about this, is that once the setting screen is merged you can rename the older boolean setting to reflect the new ui (wifi_only) and then negate EVERY usage. I think this would be an unnecessary risk to introduce bugs